css-modules/side-container.less

```diff
@@ -35,7 +37,7 @@
       display: inline-block;
       position: relative;
       list-style: none;
-      margin-bottom: -1px;
+      // margin-bottom: -1px;
```

Comment:
Do you want to keep this here or can it be removed now?

Reply:
Removed, thanks.